README.md

```diff
@@ -45,7 +46,8 @@ setTimeout(() => {
         icon: 'envelope',
         title: 'Warning Toast',
         description: 'This is a Semantic UI toast wich waits 5 seconds before closing',
-        time: 5000
+        time: 5000,
+        onClick: () => alert('you click on the toast')
```

Comment:
Let's add the passing of `onClose` as a toast option here too.

Reply:
Now this is possible without making changes to the code. Added `onClose` declaration to documentation